app/extensions/__init__.py

```diff
@@ -0,0 +1,54 @@
+# encoding: utf-8
+# pylint: disable=invalid-name,wrong-import-position,wrong-import-order
+"""
+Extensions setup
+================
+
+Extensions provide access to common resources of the application.
+
+Please, put new extension instantiations and initializations here.
+"""
+from flask_cors import CORS
+from flask_login import LoginManager
+from flask_marshmallow import Marshmallow
+from flask_restplus import Api
+from sqlalchemy_utils import force_auto_coercion
+from sqlalchemy_utils import force_instant_defaults
+
+from .flask_sqlalchemy import SQLAlchemy
+from .logging import Logging
+
+
+logging = Logging()
+
+cross_origin_resource_sharing = CORS(supports_credentials=True)
+
+db = SQLAlchemy()
+
+force_auto_coercion()
+force_instant_defaults()
```

Comment:
googled but still don't get it how `force_auto_coercion` works out.

Reply:
another copy-pasted code. I believe it have some value. https://sqlalchemy-utils.readthedocs.io/en/latest/listeners.html From the docs, it says will automatically cast data base on it specified type from SQLAlchemy's mapper.